pages/docs/core-concepts/error-handling.mdx

```diff
@@ -139,14 +144,70 @@ We could optionally also use the `Effect.catchTags` combinator to handle all err
 // @include: errors-1
 // ---cut---
 const program = pipe(
-//    ^?
+  //    ^?
   flakyFoo,
   Effect.zipRight(flakyBar),
   Effect.catchTags({
     FooError: (fooError) => Effect.succeed(`Recovering from ${fooError._tag}`),
-    BarError: (barError) => Effect.succeed(`Recovering from ${barError._tag}`)
+    BarError: (barError) => Effect.succeed(`Recovering from ${barError._tag}`),
   })
-)
+);
+```
+
+#### Modifying the error channel
```

Comment:
We're not really "modifying" the error channel, we're mapping over the error channel.

Reply:
Sorry for the confusion here, was trying to simplify the language a bit and it may have been misleading.